app/Http/Requests/StoreAlbum.php

```diff
@@ -13,7 +15,13 @@ class StoreAlbum extends FormRequest
      */
     public function authorize()
     {
-        return false;
+        if (is_null($this->user())) {
```

Comment:
Wouldn't this always be `null` with non-session based Authentication ?

Reply:
Yes, which is why we need to implement token-based API authentication and update this as appropriate.